terraform/development/shared/main.tf

```diff
@@ -59,6 +59,9 @@ locals {
   app_host                  = "review.submit-social-housing-data.levellingup.gov.uk"
   load_balancer_domain_name = "review.lb.submit-social-housing-data.levellingup.gov.uk"
 
+  new_app_host                  = "review.submit-social-housing-data.communities.gov.uk"

```

Comment:
Naming here is very temporary - we can't get rid of the old certificates yet, but the plan is once we are switched over and have a redirect in place we can remove the old ones and this code will look as it did before (just with different text for app_host and load_balancer_domain_name).